src/org/jitsi/impl/neomedia/rtcp/RTCPReceiverFeedbackTermination.java

```diff
@@ -410,20 +410,21 @@ private RawPacket doTransform(RawPacket pkt, boolean send)
             {
                 ByteArrayBuffer baf = it.next();
                 int pt = RTCPHeaderUtils.getPacketType(baf);
-                if (pt == RTCPRRPacket.RR
+                if (pt > -1
```

Comment:
if this is to handle the chrome ssrc of `1`, i think we should use a constant (like `CHROME_DEFAULT_RTCP_MEDIA_SSRC=1` and check for that specifically

Reply:
ack, just saw this is on payload type, not ssrc.  is this essentially an `isValid` type check?

Reply:
getPacketType returns an `int` (so that it has enough space to return an error value) but the RTCPFeedbackMessageEvent.PT_PS is a `byte`. I'll add a comment about that.

Reply:
i was wondering more about this new check of `pt > -1`

Reply:
`pt` is the return value of `RTCPHeaderUtils.getPacketType`, which returns -1 in case of an error.

Reply:
so if it's an error, do we want to return the packet as-is? or should we return null?